scripts/post-update.sh

```diff
@@ -307,12 +307,24 @@ if (( $PRE_0_0_19 > 0 )); then
         echo "$STRING" | sudo tee -a /boot/config.txt
     done
 
+    # Remove any configuration or commented part related to the i2c drive
+    for STRING in "i2c-dev" "i2c-bcm2708"; do
+        sudo sed -i "/$STRING/d" /etc/modules
+        echo "$STRING" | sudo tee -a /etc/modules
+    done
+
+    # Remove any console serial configuration
+    sudo sed -e 's/console=serial[0-9],[0-9]*\ //' -i /boot/cmdline.txt
+
     # Install i2c tools
     sudo apt update --yes
     sudo apt install i2c-tools --yes
 
     # Install socat to create bind between /dev/navigator and /dev/autopilot
     sudo apt install socat --yes
+
+    # Force update of mavproxy params to add mavlink2rest and ping-viewer outputs
```

Comment:
Could we just append some lines there? I don't like the idea of replacing the user's settings.

Reply:
Yes, this part should be explained in the software update instructions to avoid overwriting existing custom user configs.